lib/streamlit/runtime/scriptrunner/script_runner.py

```diff
@@ -230,9 +229,15 @@ def start(self) -> None:
         This must be called only once.
 
         """
+        # Stlite: threading is not supported on Pyodide. Use script task instead
         if self._script_task is not None:
             raise Exception("ScriptRunner was already started")
 
+        # self._script_thread = threading.Thread(
```

Comment:
nit: Is this for helping readers to understand the diffs, and do you think it's better to add these commented-out lines?
This is not an argument, but just a question, as I usually don't do this in my code relying on VCS diffs.

Reply:
A couple of other patches also have these commented-out lines, and I think it helped me a bit when doing the update. But I think using the diff of the previous update to understand what code got replaced is also sufficient. So, I think it would be fine to remove all the commented-out lines

Reply:
> A couple of other patches also have these commented-out lines

Oh, sorry for my inconsistence.

Thanks, and both work for me too. May delete them and may not 😵‍💫

Reply:
I removed all the commented out code lines 👍

Reply:
Thank you!